props/blaster.h

```diff
@@ -484,7 +499,16 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
       break;
     }
   }
-};
 
+#ifdef BLASTER_SHOTS_UNTIL_EMPTY
+  const int max_shots_ = BLASTER_SHOTS_UNTIL_EMPTY;
```

Comment:
Constants don't usually go at the bottom with the variables.

Reply:
moved.

Reply:
it doesn't seem moved...
(It should be near the top of the class.)

Reply:
> it doesn't seem moved... (It should be near the top of the class.)

I thought I moved that because I remember saying to myself "ok I'll just put it back where it was." 
Now you're saying top, so I just put it there now, up above the first function.

*EDIT* - wait. I *did* move them? I thought so. Irrelevant now, just weird that they were on the bottom again just 20 minutes ago.
https://github.com/profezzorn/ProffieOS/pull/582/commits/73c533cbded12b5f74aad403242fdbd547486023